test/src/test/java/org/corfudb/integration/MemoryFootprintIT.java

```diff
@@ -73,6 +73,9 @@ public void testCorfuTablePutMemoryFootprint() throws Exception {
                 .setStreamName("volbeat")
                 .open();
 
+        // Force GC first to prevent it from interfering with the size estimates after.
+        System.gc();
```

Comment:
Btw this is just a hint. I think sizeOf is not reliable.
I think this will just make intermittent.

Reply:
do you think we should get rid of the test itself?

Reply:
Hmm, I triggered CI a couple of times and it seems to be working. Maybe its a GC timing issue?

I remember this didnt work when we used oraclejdk, maybe openjdk handles the gc hints more reliably?

Reply:
Yes this fix did seem to make MemoryFootprintIT a bit more reliable, so can we please merge this fix so other PRs have fewer failures from this test?